api/handlers_spaces_test.go

```diff
@@ -0,0 +1,130 @@
+package api
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+// getTimeDefault
+func TestGetTimeDefault(t *testing.T) {
+	// use defaults derived in code
+	startTime, endTime, outBool := getTimeDefault()
+	if outBool {
+		t.Logf("bool of getTimeDefault output: %v\n", outBool)
+	}
+	if outBool == false {
+		t.Errorf("bool of getTimeDefault false, expected true: %v\n", outBool)
+	}
```

Comment:
this will only ever be `true` from that function

Reply:
removed timeValidity, refactored to using return errors down to base function SpaceGetHandler